### PR TITLE
Move to here

### PR DESCRIPTION
**Pull request by scarabus**

### CODE REVIEW

File: Intro01/Reflex/src/App.rfx [hunk 1]
```diff
@@ -0,0 +1,246 @@
+SERIES_AUTHORITY="datacapture";
```
Comment (jonathan-major) [inline Intro01/Reflex/src/App.rfx:1]: can you add an overall description of the scripts intent?

File: Intro01/Reflex/src/App.rfx [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+println("\nDone\n");
+
+// **********************
+// Get data from the blob using the pull operation
+// **********************
+
+println("Retrieving raw CSV content from " + rawCsvUri);
+
+// Pull data from the uploaded Blob.
+// Rapture keeps track of the types of blobs.
```
Comment (jonathan-major) [inline Intro01/Reflex/src/App.rfx:58]: add list of supported blob types

File: Intro01/Reflex/src/App.rfx (hunk 1 above)
Comment (scarabus) [inline Intro01/Reflex/src/App.rfx:1]: I will, but isn't that better served by the documentation?
https://incapture.atlassian.net/wiki/display/PDT/Rapture+Getting+Started%3A+REFLEX

File: Intro01/Reflex/src/App.rfx (hunk 2 above)
Comment (scarabus) [inline Intro01/Reflex/src/App.rfx:58]: Again, better served in the tutorial documentation. (Certainly isn't documented anywhere else)

**Pull request merged**